raiden/blockchain/events.py

```diff
@@ -144,7 +147,7 @@ def get_all_netting_channel_events(
         pyethapp_chain,
         netting_channel_address,
         events=ALL_EVENTS,
-        from_block=0,
+        from_block='earliest',
```

Comment:
This documentation is inconsistent with the actual behavior. It was `latest` before, and it filtered out the events. With `earliest`, for what I've seen, it is equivalent as `from_block=0`, getting all events since ever, that's why I changed it, it was the actual source of the bug #808